openmdao/components/linear_system.py

```diff
@@ -6,8 +6,22 @@
 
 
 class LinearSystem(Component):
-    """ A component that solves a linear system Ax=b where A and x are params
-    and x is a state."""
+    """
+    A component that solves a linear system Ax=b where A and x are params
+    and x is a state.
+
+    Options
+	-------
```

Comment:
OK, replaced embedded tabs with spaces, regenerated docstrings, re-tested, rebuilt and inspected docs, updated test. All appears well.